addons/stock/tests/test_robustness.py

```diff
@@ -245,3 +246,50 @@ def test_lot_quantity_remains_unchanged_after_done(self):
         moveA._set_lot_ids()
 
         self.assertEqual(moveA.quantity, 5)
+
+    def test_new_move_done_picking(self):
+        """ Ensure that adding a Draft move to a Done picking doesn't change the picking state
+        """
+        categ_id = self.env.ref('product.product_category_all').id
```

Comment:
product category was made not required in saas-18.1 by https://github.com/odoo/odoo/commit/3931aea9c017b1a85992b0f147f65b3df12b921a so I think you can either just drop it or make it `product.product_category_goods` (the replacement for the all category)